experiment/build/builder.py

```diff
@@ -50,6 +51,27 @@
 logger = logs.Logger('builder')  # pylint: disable=invalid-name
 
 
+def get_fuzzer_benchmark_pairs(fuzzers, benchmarks):
+    """Return a tuple of (fuzzer, benchmark) pairs to build. Excludes
+    blacklisted fuzzers from each benchmark.
```

Comment:
nit: consider using "skip list" instead here and everywhere else.

Reply:
Done!

Reply:
thanks!